src/Entity/EntityFactory.php

```diff
@@ -103,6 +104,14 @@ public static function createBroadcast(array $data = []): Broadcast
         return $entity;
     }
 
+    public static function createMessage(array $data = []): Message
+    {
+        /* @var $entity Message */
+        $entity = self::create('Message', $data);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$entity". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1488)